packages/react-native-reanimated/Common/cpp/worklets/WorkletRuntime/WorkletRuntime.cpp

```diff
@@ -73,8 +78,7 @@ WorkletRuntime::WorkletRuntime(
   WorkletRuntimeCollector::install(rt);
   WorkletRuntimeDecorator::decorate(rt, name, jsScheduler);
 
-  auto codeBuffer = std::make_shared<const jsi::StringBuffer>(
-      "(" + valueUnpackerCode + "\n)");
+  auto codeBuffer = std::make_shared<const jsi::StringBuffer>(getJsValueUnpackerString());
```

Comment:
The initial attack vector for prototype pollution seems mitigated here, since the value unpacker has been hardcoded and there was no more usage of `asString` anymore in the original source (there was in 2.15.0).

I think we can get rid of all the symbol hardening stuff if that's the case.